extensions/ql-vscode/src/model-editor/modeled-method-fs.ts

```diff
@@ -94,7 +88,11 @@ export async function loadModeledMethods(
   );
   for (const modeledMethods of Object.values(modeledMethodsByFile)) {
     for (const [key, value] of Object.entries(modeledMethods)) {
-      existingModeledMethods[key] = value;
+      if (!(key in existingModeledMethods)) {
+        existingModeledMethods[key] = value;
+      }
+
+      existingModeledMethods[key].push(...value);
```

Comment:
This looks like it's going to duplicate the modeled methods.

Not sure if it was intended to be:
```suggestion
      if (!(key in existingModeledMethods)) {
        existingModeledMethods[key] = value;
      } else {
        existingModeledMethods[key].push(...value);
      }
```

or:
```suggestion
      if (!(key in existingModeledMethods)) {
        existingModeledMethods[key] = [];
      }

      existingModeledMethods[key].push(...value);
```